airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -95,9 +106,8 @@ public JsonNode toJdbcConfig(final JsonNode config) {
     // and can't
     // remove zero date values.
     // since zero dates are placeholders, we convert them to null by default
-    jdbcUrl.append("?zeroDateTimeBehavior=convertToNull");
     if (!additionalParameters.isEmpty()) {
-      jdbcUrl.append("&");
+      jdbcUrl.append("?");
       additionalParameters.forEach(x -> jdbcUrl.append(x).append("&"));
```

Comment:
This statement leaves an `&` at the end. You can use the `String#join` method to eliminate this problem.

```suggestion
      jdbcUrl.append(String.join("&", additionalParameters);
```

The unit tests need to be adjusted accordingly.